packages/integration-tests/tests/config.ts

```diff
@@ -0,0 +1,25 @@
+import {Page} from '@playwright/test'
+
+type TestingMode = 'production' | 'local'
+
+export class Config {
+  readonly testingMode: TestingMode
+  readonly rootUrlPath: string
+
+  constructor(env = process.env) {
+    this.testingMode = env.INTEGRATION_TESTING_MODE === 'production' ? 'production' : 'local'
+    this.rootUrlPath =
+      this.testingMode === 'production' ? 'https://action.parabol.co' : 'http://localhost:3000'
```

Comment:
We might not use the production-testing mode of this straight away, but I had discussed this with @mattkrick and @jordanh during our intro calls, so I thought I'd demo it as part of the prototype. My thought is that it's easier to use this `Config` object from the get-go so we have flexibility in the future.

Reply:
Oh, another thing I just thought about is that this could be useful for smoke testing for customers who have on-prem deployments (cc @adaniels-parabol)

Reply:
That would be great @blimmer I currently do PPMI deployments in gitlab. It would be great to have a gitlab pipeline I could just put a run var in at the time of the deployment and test. Triggered via a webhook to the pipeline <3